frontend/tool/demo-server.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+process.env.NODE_ENV = 'demo'; // this is the only diff from dev server
+let devPort = 8999;
+let opn = require('opn');
+let express = require('express');
+let webpack = require('webpack');
+let proxyMiddleware = require('http-proxy-middleware');
+let webpackConfig = require('./webpack.dev.config');
```

Comment:
Here is loading the incorrect config. Should update `./webpack.dev.config` to `./webpack.demo.config`